### PR TITLE
feat: support for redwood

### PR DESCRIPTION
**Pull request by MoisesGSalas**
# Description

1. Redwood support: this was only bumping the versions.
2. When using `MFE_EXTENSIONS_BY_PATH` course-authoring in the studio domain and the rest of the core MFEs in the lms domain. For any custom/additional (e.g. [ecommerce](https://github.com/overhangio/tutor-ecommerce/blob/6777742f0d51860ed878e374978730c8f4041cac/tutorecommerce/plugin.py#L74-L82)) MFE use the `MFE_HOST` domain.
3. Point the MFEs urls to the LMS/CMS. The originals are taken from the [`tutor-mfe`](https://github.com/eduNEXT/tutor-mfe/blob/f7f21edf444ba581fc9bc2c01e785278b874175b/tutormfe/patches/openedx-lms-production-settings).
4. Add some additional CORS rules (for studio while using course-authoring) and remove some paragon configuration that is only useful in our forks of each MFE.

### CODE REVIEW

File: tutormfe_extensions/patches/openedx-lms-production-settings [hunk 1]
```diff
@@ -1,16 +1,46 @@
 {% if MFE_EXTENSIONS_BY_PATH %}
 LEARNING_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/learning"
+MFE_CONFIG["LEARNING_BASE_URL"] = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/learning"
+
+{%- if get_mfe("authn") %}
+AUTHN_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/authn"
+AUTHN_MICROFRONTEND_DOMAIN  = "local.edly.io/authn"
```
Comment (Henrrypg) [inline tutormfe_extensions/patches/openedx-lms-production-settings:7]: This should be LMS_HOST too, rigth?

File: tutormfe_extensions/patches/openedx-lms-production-settings [hunk 2]
```diff
@@ -1,16 +1,46 @@
...
+
+{% if get_mfe("communications") %}
+COMMUNICATIONS_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/communications"
+{% endif %}
+
+{% if get_mfe("course-authoring") %}
+MFE_CONFIG["COURSE_AUTHORING_MICROFRONTEND_URL"] = "{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ LMS_HOST }}/course-authoring"
+{% endif %}
+
+CORS_ORIGIN_WHITELIST.append("http://studio.local.edly.io")
```
Comment (Henrrypg) [inline tutormfe_extensions/patches/openedx-lms-production-settings:45]: This need to be pointed to CORS_ORIGIN_WHITELIST.append("{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ CMS_HOST }}") ?

Comment (MoisesGSalas) [reply tutormfe_extensions/patches/openedx-lms-production-settings]: fixed

File: tutormfe_extensions/patches/openedx-lms-production-settings (hunk 1 above)
Comment (MoisesGSalas) [reply tutormfe_extensions/patches/openedx-lms-production-settings]: fixed

**Review by Henrrypg — APPROVED** [review]

**Pull request merged**